ip/draft-ietf-detnet-ip.xml

```diff
@@ -838,7 +838,7 @@ DetNet                    |L2/SbN|          |L2/SbN|
               when processing IPv6 packets. This includes 
               the next protocol
               values defined in <xref target="nxt-proto"/> and other
-              non-zero values.
+              values, including zero.
```

Comment:
I'm not sure I understand the text went from next-protocol values for  TCP, UDP and IPsec  but not zero  and is changed to  TCP, UDP  and IPsec and Zero?